staging/src/k8s.io/apiserver/go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/fsnotify/fsnotify v1.5.4
 	github.com/gogo/protobuf v1.3.2
+	github.com/google/cel-go v0.12.5
```

Comment:
No need to worry about this dep. There is ongoing effort to move CEL-related code to `k8s.io/apiserver` anyways.